docs/api/main-config-preview-annotations.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'config.previewAnnotations'
+---
+
+Type: `string[] | ((config: string[], options: Options) => string[] | Promise<string[]>)`
+
+Add additional scripts to run in the story preview.
+
+<div class="aside">
+
+💡 Mostly used by [addons](../addons/writing-presets.md) and [frameworks](../contribute/framework.md#previewjs-example). Storybook users should likely add scripts to [`preview.js`](../configure/overview.md#configure-story-rendering) instead.
+
+</div>
+
+```ts
+// @storybook/nextjs framework's src/preset.ts
```

Comment:
This is the only example where it's not part of `main.ts`, because it didn't seem like it would be used that way. Is that correct?